js/main.js

```diff
@@ -0,0 +1,78 @@
+const PHOTO_COUNT = 25;
+const COMMENT_COUNT = 30;
+
+const Likes = {
+  MIN: 15,
+  MAX: 200
+};
+
+const Avatars = {
+  MIN: 1,
+  MAX: 6
+};
+
+const messages = [
```

Comment:
Тоже КОНСТАНТА по сути
```suggestion
const MESSAGES = [
```